src/Logging/BuildDBLogHandler.php

```diff
@@ -71,7 +71,7 @@ protected function write(array $record)
     {
         $message = (string)$record['message'];
         $message = str_replace(['\/', '//'], '/', $message);
-        $message = str_replace($this->build->getBuildPath(), '<BUILD_PATH>/', $message);
+        //$message = str_replace($this->build->getBuildPath(), '<BUILD_PATH>/', $message);
```

Comment:
Why?

Reply:
Had trouble setting the build path correctly but found the trouble was in how I set it in the config, not in the code.   I'll remove the comment line.